p2p/upnp/probe.go

```diff
@@ -18,26 +17,26 @@ type UPNPCapabilities struct {
 func makeUPNPListener(intPort int, extPort int, logger log.Logger) (NAT, net.Listener, net.IP, error) {
 	nat, err := Discover()
 	if err != nil {
-		return nil, nil, nil, errors.New(fmt.Sprintf("NAT upnp could not be discovered: %v", err))
+		return nil, nil, nil, fmt.Errorf("NAT upnp could not be discovered: %v", err)
```

Comment:
`errors.Errorf` throws as undefined. The official documentation (of the `errors` package) recommends using `fmt.Errorf` for errors that need formatting